filebeat/tests/system/test_crawler.py

```diff
@@ -528,7 +528,7 @@ def test_utf8(self):
                 lambda: self.output_has(lines=1), max_timeout=10)
 
         # Append utf-8 chars to check if it keeps reading
-        with codecs.open(testfile, "a") as f:
+        with codecs.open(testfile, "a", "utf-8") as f:
```

Comment:
`utf_8` is the name of the codec in Python. `utf-8` is an alias name. I think I used `utf_8` in other places. So to prevent having a mix of "utf-8" and "utf_8" can you change this to `utf_8`.